build.psm1

```diff
@@ -158,7 +158,30 @@ function Start-PSBuild {
     }
 
     # save Git description to file for PowerShell to include in PSVersionTable
-    git --git-dir="$PSScriptRoot/.git" describe --dirty --abbrev=60 > "$psscriptroot/powershell.version"
+    $powershellVersion = git --git-dir="$PSScriptRoot/.git" describe --dirty --abbrev=60
+    $powershellVersion > "$PSScriptRoot/powershell.version"
```

Comment:
I don't see that file being used in any tests. i do see the string `powershell.version` in a test, but it sort of looks like a broken test to me.